__tests__/licenses.test.ts

```diff
@@ -30,7 +29,7 @@ const rubyChange: Change = {
   name: 'actionsomething',
   version: '3.2.0',
   package_url: 'pkg:gem/actionsomething@3.2.0',
-  license: 'BSD',
+  license: 'BSD-3-Clause',
```

Comment:
Just noting this here—`BSD` is not a valid license type in earlier versions of this action either, but tests used to bypass that check.